var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,7 +29,6 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
```

Comment:
This package needs the line:
```
depends_on('python@2:3.3')
```
@adamjstewart did I get the syntax right for that?